packages/frontend/src/components/EditorRightDrawer/Step.tsx

```diff
@@ -90,9 +113,7 @@ export default function Step(props: StepProps): React.ReactElement | null {
               (substep) =>
                 substep.key &&
                 // NOTE: webhook trigger is a special case where we want to show the step configuration immediately
-                ((step.appKey === 'webhook' && step?.webhookUrl) ||
-                  ['chooseConnection', 'testStep'].includes(substep.key) ===
-                    false) && (
+                shouldShowSubstep(substep) && (
```

Comment:
To deconflict with MRF PRs

```suggestion
                 ((step.appKey === 'webhook' && step?.webhookUrl) ||
                  substep.key !== 'chooseConnection') &&
```